### DISCUSSION TITLE
Copy HTTPS URL Keyboard Shortcut

### DISCUSSION THREAD

**Discussion by countstoneula** [iterate-ch/cyberduck#15215] [Q&A]
I find myself needing to copy the HTTPS URL within Cyberduck almost constantly. I probably did it about 3000 times today. But the keyboard "Command+C" function is setup to copy the file rather than the URL. There is a "Shift+Command+C" that copies the FTP URL but nothing for the Copying of the HTTPS URL that I require.

Does anyone know if there is a way to assign or reassign these quick keyboard commands? I never really need to use the other two. Or is there a keyboard command for copying this HTTPS URL that I don't know about? Even if it's a large combination I can reassign one of my Function keys to make it work.

Thanks.

**Comment by dkocher** [comment]
Refer to *Create keyboard shortcuts for apps on Mac* [^1] to create a custom keyboard shortcut.

[^1]: https://support.apple.com/en-lamr/guide/mac-help/mchlp2271/mac

**Reply by countstoneula** [reply]
Thanks! I just changed Copy to "option+c" and made HTTPS URL "command+c".
Thank you so much This is going to save me so much time!
